test/simulation/README.md

```diff
@@ -0,0 +1,46 @@
+# xud-simulation
+
+xud network simulation framework and integration tests.
+
+## Installation & Usage
+
+This project uses [go modules](https://github.com/golang/go/wiki/Modules) 
+    to manage dependencies.
+    
+Requirements:
+
+* Golang version >= 1.11
+* Git
+
+Installation & Usage:
+
+```bash
+$ GO111MODULE=on go test -v
```

Comment:
Does it make sense to change this to `npm run test:sim`?

Reply:
I didn't do it so it will be treated as independent folder to be run directly. But since we want to create a dependency with the `.proto` files, I can that.

Reply:
I fixed that, but didn't push yet because I wanted you to see the current travis failure status.